lisa/messages.py

```diff
@@ -60,16 +91,7 @@ class PerfMessage(MessageBase):
 
 @dataclass
 class DiskPerformanceMessage(PerfMessage):
-    tool: str = constants.DISK_PERFORMANCE_TOOL
-    test_case_name: str = ""
-    platform: str = ""
-    location: str = ""
-    host_version: str = ""
-    guest_os_type: str = "Linux"
-    distro_version: str = ""
-    vmsize: str = ""
-    kernel_version: str = ""
-    lis_version: str = ""
+    tool: str = constants.DISK_PERFORMANCE_TOOL_FIO
```

Comment:
leave it without value like below, so evey time it should be filled.
tool: str

Reply:
updated